src/bootstrap/flags.rs

```diff
@@ -134,7 +134,7 @@ pub enum Subcommand {
         paths: Vec<PathBuf>,
     },
     Setup {
-        profile: Profile,
+        paths: Vec<PathBuf>,
```

Comment:
This doesn't look quite right. Setup should never be run more than once; if it does, it will give a hard error because there's already an existing config.toml. Can you change this to a single path?

Reply:
some changes were reverted following this comment https://github.com/rust-lang/rust/pull/96584#issuecomment-1221323383 , I have changed back to single path, please see the new commit :)